test/common.mak

```diff
@@ -14,11 +14,13 @@ SRC:=src
 GENERATED:=./generated
 ROOT:=$(GENERATED)/$(OS)/$(BUILD)/$(MODEL)
 
+OPTIONAL_PIC:=$(if $(PIC),-fPIC,)
```

Comment:
I thought that I had done what was requested, but I could have misunderstood, and my makefile-foo is quite weak.